inventory/environ.py

```diff
@@ -96,15 +93,28 @@ def getDefaultVars():
     defaultVars = loadDefaultSplunkVariables()
     overrideEnvironmentVars(defaultVars)
 
+    getIndexerClustering(defaultVars)
+    getSearchHeadClustering(defaultVars)
+    getMultisite(defaultVars)
+    getSplunkWebSSL(defaultVars)
     getDistributedTopology(defaultVars)
     getLicenses(defaultVars)
     defaultVars["splunk"]["role"] = os.environ.get('SPLUNK_ROLE', 'splunk_standalone')
-    defaultVars["splunk_home_ownership_enforcement"] = False if os.environ.get('SPLUNK_HOME_OWNERSHIP_ENFORCEMENT', "").lower() == "false" else True
```

Comment:
Made these inline sections slightly more verbose because 1) pylint gave me some warnings about line length and 2) it is a little hard to read.

Maybe this could get moved around in the future for simpler testing as well.